easypost/easypost_object.py

```diff
@@ -180,18 +185,18 @@ def __repr__(self):
 
         return "<%s%s at %s> JSON: %s" % (type(self).__name__, type_string, hex(id(self)), json_string)
 
-    def __str__(self):
+    def __str__(self) -> str:
         return self.to_json(indent=2)
 
-    def __eq__(self, other):
+    def __eq__(self, other) -> bool:
         if not isinstance(other, EasyPostObject):
             return False
         return self.__str__() == other.__str__()
 
-    def to_json(self, indent=None):
+    def to_json(self, indent=None) -> str:
         return json.dumps(self.to_dict(), sort_keys=True, indent=indent, cls=EasyPostObjectEncoder)
 
-    def to_dict(self):
+    def to_dict(self) -> dict:
```

Comment:
This must be `Dict` and not `dict`, I'm not sure how this didn't fail on CI to be honest...